inst/htmlwidgets/datatables.js

```diff
@@ -50,7 +50,16 @@ var transposeArray2D = function(a) {
 HTMLWidgets.widget({
   name: "datatables",
   type: "output",
-  renderValue: function(el, data) {
+  initialize: function(el, width, height) {
+    $(el).text('TO BE RENDERED...');
```

Comment:
Can it at least be a nonbreaking space, or an empty div, or something else that wouldn't be so obviously visible?

Other than that, LGTM.